app/Actions/UserList/DeleteUserListAction.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Hedonist\Actions\UserList;
+
+use Hedonist\Repositories\UserList\UserListRepository;
+
+class DeleteUserListAction
+{
+    private $userListRepository;
+
+    public function __construct(UserListRepository $userListRepository)
+    {
+        $this->userListRepository = $userListRepository;
+    }
+
+    public function execute(DeleteUserListRequest $request): DeleteUserListResponse
+    {
+        $this->userListRepository->deleteById($request->getId());
+        return new DeleteUserListResponse();
```

Comment:
One enter before return statement is necessary if body of method doesn't consist of only one return statement